Neos.Flow/Classes/Security/Authentication/AccountWithoutPersistence.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Neos\Flow\Security\Authentication;
+
+use Neos\Flow\Security\Account;
+
+/**
+ * This is an account, which is no entity - useful to be used in sessionless tokens, or tokens, which can create
+ * their own Account
+ */
+class AccountWithoutPersistence extends Account
```

Comment:
hmm i dont understand this 😂 _why_ is it without persistence? doesnt it just inherit?

If the change here and the changed `FileBasedSimpleKeyProvider` is only to fix the setup _(a pseudo fix, because then it will fail at another point - sorry for being so pessimistic but im spending a lot time into the new alternative https://github.com/neos/neos-development-collection/issues/4243 so i just want it to die)_ i rather have the code in the neos/setup package with an extended `FileBasedSimpleKeyProvider` so i can burn the code with fire once the new setup is ready.

Putting code like this `AccountWithoutPersistence` directly into flow will never allow us to get rid of it ^^ as its seemingly api no? - also we wont likely going to clean it up after the old setup is finally dead.

Reply:
Is this the `TransientAccount` that we initially created here ??

https://github.com/neos/flow-development-collection/pull/1939/files#diff-5c52a2da4d71ad6dbcb5cdc38d63b507fd04199d0790a65c074ad038eb11dc03

This patch also creates a solution for the FileBasedSimplyKeyProvider.

I will love that we solve the AccountInterface task before creating solution around the real issue at hand

Reply:
@sorenmalling : We won't merge this in "past context" - so in 7.x or 8.x. Yes the branch would be MUCH better to have merged. But as it touches many places and is at least at one place breaking if I see it correctly, we simply cannot merge it in released versions.

So our goal is to have it in as soon as possible - ideally 9.0, 9.1 at latest.

But this won't fix the problems already "out".

Reply:
Late to the game, but two things:
- isn't this class gonna receive a schema still, since it inherits from an entity?
- as far as I understood, this is supposed to be a hotfix intermediate class, that will vanish or be replaced. Let's do everything to make this super clear and prevent anyone from using the class outside the FileBasedSimpleKeyProvider. So make the name annoyingly explicit about it, and add a doc block stating this, potentially also directly annotating it as `@deprecated`

Reply:
what about using an anonymous class? Would that work? (Ignore my comment if its totally useless 😂 )

Reply:
I have already tried that:
> Unfortunately we cannot use anonymous classes - I thought the idea would be nice, as it would allow us to get around this trouble without creating new classes... But that won't do, as anonymous classes cannot be serialized, which is crucial for the session to work.

Reply:
@albe: 
- No it becomes no schema, because the Entity annotation does not "inherit".
- Yes, if we think this "approach" is acceptable at all, we relly should make clear, that this is only an "interim" solution. And we really should take care to merge the better account semantics at least for 9.0